swagger.yml

```diff
@@ -22,19 +18,26 @@ definitions:
         type: string
       birthday:
         type: string
-      old_password:
-        type: string
-      password:
-        type: string
-      password_r:
-        type: string
       profile_picture:
         type: string
       points:
         type: integer
         format: int32
       date_created:
+        type: string 
+    allOf:
```

Comment:
Instead of **allOf**, a passwords property of Passwords migh be more suitable.
```
passwords:
    type: object
    $ref: '#/definitions/Passwords'
```